### PR TITLE
VOTE-2005: PX Homepage Refinements

### PR DESCRIPTION
**Pull request by mlloydbixal**
## Jira ticket

[VOTE-2075](https://cm-jira.usa.gov/browse/VOTE-2075)
[VOTE-2005](https://cm-jira.usa.gov/browse/VOTE-2005)

## Description

VOTE-2075:
-Updates button color to fix contrast issue. I made the change globally and included updates to the active and hover states of ac-warm. Active and hover do match per Craig's guidance so there is no visual change.

VOTE-2005:
-Updates the homepage hero design
-wraps button in hero with a div instead of paragraph tag so paragraph styles don't get applied
*https://www.figma.com/design/u82ICmmHNBKpE1Tw4nsO0K/Vote.gov-Redesign-2024-(Master)?node-id=9007-30880&m=dev

### Post-deploy steps

1. lando retune
2. npm run build

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [x] The file changes are relevant to the task objective.
- [x] Code is readable and includes appropriate commenting.
- [x] Code standards and best practices are followed.
- [x] QA/Test steps were successfully completed, if applicable.
- [x] Applicable logs are free of errors.

### CODE REVIEW

File: web/themes/custom/votegov/src/sass/components/hero.scss [hunk 1]
```diff
@@ -57,11 +58,27 @@
...
+
+  .usa-button {
+    font-family: $theme-font-type-sans;
+    font-size: 16x;
+    width: auto;
+
+    @include at-media-max('mobile') {
+      width: 193px;
+    }
+  }
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/components/hero.scss:70]: I don't understand why any of these styles are necessary can you describe why they are needed. I'm not seeing the affects.

File: web/themes/custom/votegov/src/sass/components/hero.scss [hunk 2]
```diff
@@ -20,9 +20,10 @@
 }
 
 .vote-hero__image {
+  @include u-order(2);
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/components/hero.scss:23]: If the hero image is always expected to be below on mobile then we need to update the paragraph template so the image comes second in the dom and we can remove this reordering altogether. Then we can make sure we are only setting the container to flex on tablet-lg breakpoint and above using mobile-first approach.

File: web/themes/custom/votegov/src/sass/components/hero.scss [hunk 3]
```diff
@@ -57,11 +58,27 @@
...
+      width: 193px;
+    }
+  }
+
+  p {
+    @include u-margin-top(2);
+    @include u-margin-bottom(3);
+    font-family: $theme-font-type-sans;
+    font-size: 19.2px;
+  }
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/components/hero.scss:77]: Create a new wrapper element around the field body print statement in the template with a class of "vote-hero__text usa-intro". Make sure it's wrapped in a conditional so it won't print if the body field is empty. Then replace this style with ` > * + *` and apply `@include u-margin-top(3);`

File: web/themes/custom/votegov/src/sass/components/hero.scss [hunk 4]
```diff
@@ -57,11 +58,27 @@
...
 }
 
 .vote-hero__heading {
-  @include u-margin-bottom(4);
-  @include u-font-size('body', 12);
+  @include u-font('serif', 'xl');
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/components/hero.scss:81]: replace size unit `'xl'` with 12. Use numeric units from USWDS.

File: web/themes/custom/votegov/src/sass/_variables.scss [hunk 5]
```diff
@@ -6,9 +6,9 @@ $base-primary-hover: #2E598A; // blue-60
 $base-primary-active: $base-dark;
 
 // Accent Colors.
-$ac-warm: #CB5154; // red-cool-50
-$ac-warm-hover: #A13032;
-$ac-warm-active: #721113;
+$ac-warm: #992E30;
+$ac-warm-hover: #6D1516;
+$ac-warm-active: #6D1516;
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/_variables.scss:11]: delete the `$ac-warm-active` color variable and replace any use of it in scss with `$ac-warm-hover`

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: web/themes/custom/votegov/src/sass/components/hero.scss (hunk 1 above)
Comment (mlloydbixal) [reply web/themes/custom/votegov/src/sass/components/hero.scss]: You're right I don't think this is necessary since the buttons are styled globally

**Review by rayestrada — APPROVED** [review]

**Pull request merged**